src/providers/ipa/ipa_s2n_exop.c

```diff
@@ -979,6 +979,19 @@ static errno_t s2n_response_to_attrs(TALLOC_CTX *mem_ctx,
     return ret;
 }
 
+static const char *ipa_s2n_reqtype2str(enum request_types request_type)
+{
+    if (request_type == REQ_SIMPLE) {
+        return "REQ_SIMPLE";
+    } else if (request_type == REQ_FULL) {
+        return "REQ_FULL";
+    } else if (request_type == REQ_FULL_WITH_MEMBERS) {
+        return "REQ_FULL_WITH_MEMBERS";
+    } else {
+        return "Unknown request type";
+    }
```

Comment:
Sumit requested few changes. :-) I will request another tiny change as well.

Could you use `switch` / `case` here?